src/index.html

```diff
@@ -0,0 +1,210 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" media="screen" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+    <header class="header">
+        <div class="header-container">
+            <nav class="nav-item1"><a class="Mylogo" href="">My Logo</a></nav>
+            <nav class="nav-item2"></nav>
+            <nav class="nav-item3">
+                <img class="toggle" src="../assets/toggle.png" alt="toggle" onclick="openNav()">
+            </nav>
+            <div id="myNav" class="overlay">
+                <a class="closebtn"> &times; </a>
+                <div class="overlay-content">
+                    <a class="pop-nav" href="#portfolio">Portfolio</a>
+                    <a class="pop-nav" href="#about">About</a>
+                    <a class="pop-nav" href="#contact">Contact</a>
+                </div>
+            </div>
+        </div>
+        <nav class="web-nav">
+            <div class="logo">
+                <a class="Mylogo" href="">My Logo</a>
+            </div>
+            <ul>
+                <li><a href="#portfolio">Portfolio</a></li>
+                <li><a href="#about">About</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
+        </nav>
+    </header>
+    <section class="content">
+        <h1 class="intro">I’m Daniel Glad to see you!</h1>
+        <p class="intro1">
+            I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <a class="intro2">LET’S CONNECT</a>
+        <ul class="social">
+            <li class="social-icon twitter">
+                <a><img src="../assets/twitter.png" alt="twitter"></a>
+            </li>
+            <li class="social-icon linkedin">
+                <a><img src="../assets/Linkedin.png" alt="linkedin"></a>
+            </li>
+            <li class="social-icon">
+                <a><img src="../assets/m.png" alt="m"></a>
+            </li>
+            <li class="social-icon github">
+                <a><img src="../assets/github.png" alt="github"></a>
+            </li>
+            <li class="social-icon fingers">
+                <a><img src="../assets/fingers.png" alt="fingers"></a>
+            </li>
+        </ul>
+    </section>
+    <section id="portfolio" class="project-section">
+
+    </section>
+    <section id="about" class="about">
+        <div class="left">
+            <h2 class="about-me"> About Myself</h2>
```

Comment:
Well done for fixing the right margin issue as requested by the previous reviewer. You seem to have introduced and issue where there is much of an overflow on the page. Kindly fix this too.


Your design:
![image](https://user-images.githubusercontent.com/20554583/219932258-bd4c1028-d07c-49a7-8427-6c7899813143.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/219932303-ded211fb-0f6b-4996-8ab7-944f2563f12b.png)

Reply:
Sorry, I don´t understand